features/FFF_CFE_Integration_V4/NP-full-1-V5-2.feature

```diff
@@ -3,38 +3,20 @@ Feature:
 
     Scenario: Test that the correct output is produced for the following set of data.
         Given I am undertaking a certificated assessment
-        And A submission date of "2021-05-10"
-        And I add the following applicant details for the current assessment:
-            | date_of_birth               | 1979-12-20 |
-            | involvement_type            | applicant  |
-            | has_partner_opponent        | false      |
-            | receives_qualifying_benefit | false      |
         And I add the following proceeding types in the current assessment:
             | ccms_code | client_involvement_type |
             | DA001     | A                       |
             | SE013     | A                       |
             | SE003     | A                       |
-        And I add the following dependent details for the current assessment:
-            | date_of_birth | in_full_time_education | relationship   | monthly_income | assets_value |
-            | 2018-12-20    | FALSE                  | child_relative | 0.00           | 0.00         |
-        And I add the following other_income details for "friends_or_family" in the current assessment:
-            | date       | client_id | amount |
-            | 2021-05-10 | id3       | 100.00 |
-            | 2021-04-10 | id2       | 100.00 |
-            | 2021-03-10 | id3       | 100.00 |
+        And I have a dependant aged 2
+        And I add other income "friends_or_family" of 100 per month
         And I add the following irregular_income details in the current assessment:
             | income_type  | frequency | amount |
             | student_loan | annual    | 120.00 |
-        And I add the following outgoing details for "rent_or_mortgage" in the current assessment:
-            | payment_date | housing_cost_type | client_id | amount |
-            | 2021-05-10   | rent              | id7       | 10.00  |
-            | 2021-04-10   | rent              | id8       | 10.00  |
-            | 2021-03-10   | rent              | id9       | 10.00  |
-        And I add the following capital details for "bank_accounts" in the current assessment:
-            | description | value  |
-            | Bank acc 1  | 4999.0 |
-            | Bank acc 2  | 0      |
-            | Bank acc 3  | 0      |
+        And I add outgoing details for "rent_or_mortgage" of 10 per month
+        And I add 4999 capital of type "bank_accounts"
+        And I add 0 capital of type "bank_accounts"
+        And I add 0 capital of type "bank_accounts"
```

Comment:
Is this a duplicate line or is it testing something specific?

Reply:
I've no idea - its just a port of the above line of 1 bank account plus.2 empty ones